onadata/apps/api/management/commands/delete_users.py

```diff
@@ -0,0 +1,102 @@
+"""
+Delete users management command.
+"""
+import sys
+from django.contrib.auth.models import User
+from onadata.apps.logger.models import XForm
+from onadata.apps.logger.models import Instance
+from onadata.apps.logger.models import Project
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+
+
+def get_user_object_stats(username):  # pylint: disable=R0201
+    """
+    Get User information.
+    """
+    user = User.objects.get(username=username)
+    # Get the number of projects for this user
+    user_projects = Project.objects.filter(created_by=user).count()
+    # Get the number of forms
+    user_forms = XForm.objects.filter(user=user).count()
+    # Get the number of submissions
+    user_sumbissions = Instance.objects.filter(user=user).count()
+
+    result = {
+        'projects': user_projects,
+        'forms': user_forms,
+        'submissions': user_sumbissions
+    }
+
+    return result
+
+
+def inactivate_user(username):
+    """
+    Soft deletes the user termporarily.
+    """
+    try:
+        user = User.objects.get(username=username)
+        # set inactive status on user account
+        user.is_active = False
+        # append a timestamped suffix to the username
+        # to make the initial username available
+        soft_deletion_time = timezone.now()
+        deletion_suffix = soft_deletion_time.strftime('-deleted-at-%s')
+        user.username += deletion_suffix
+
+        user.save()
+        sys.stdout.write(
+            sys.style.SUCCESS(
+                'User {} deleted successfully.'.format(username)))
```

Comment:
Would be nice if this message gave the summary of objects deleted for that user. `User DavisRayM deleted successfully. User has 2 XForms... etc.`

Reply:
Really? I thought this information was relevant just before you delete the user.
Why would someone need to see this again after they had deleted the account?

Reply:
In case they don't set `user_input` to showcase the data.... might not be important.... yea not sure this is up to you honestly. I thought it would be nice to just have it just in case